app/components/commons/SocialSignInButtons.jsx

```diff
@@ -0,0 +1,63 @@
+import {React} from 'react';
+import CustomButton from './CustomButton'
+import { GoogleSignin, statusCodes } from '@react-native-google-signin/google-signin';
+import { useEffect } from 'react';
+import {PostLoginGoogle} from '../../utils/apicalls/ApiCalls'
+import {useNavigation} from '@react-navigation/native'
+
+export default SocialSignInButtons = () => {
+    
+    const onSignInFacebook = () => {
+        console.warn('facebook');
+    }
+
+    const navigation = useNavigation()
+
+    useEffect( () => {
+        GoogleSignin.configure({
+            androidClientId: '310650990221-v7rjbq7joe78ebbcs47v5ur28m8o37q1.apps.googleusercontent.com'
```

Comment:
Esto es info sensible? Ni idea que es.

Reply:
Eso es un numero de cliente para poder hacer el sign in por Google, lo movi a un archivo de configuracion, creo que es lo mejor